zerver/views/invite.py

```diff
@@ -232,6 +233,46 @@ def resend_user_invite_email(
     return json_success(request)
 
 
+@require_member_or_admin
+@typed_endpoint
+def edit_multiuse_invite(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    *,
+    invite_id: PathOnly[int],
+    stream_ids: Json[list[int]] | None = None,
+    invite_as: Annotated[
+        Json[int] | None,
+        check_int_in_validator(list(PreregistrationUser.INVITE_AS.values())),
+    ] = None,
+    include_realm_default_subscriptions: Json[bool] | None = None,
+) -> HttpResponse:
+    try:
+        invite = MultiuseInvite.objects.get(id=invite_id)
+    except MultiuseInvite.DoesNotExist:
```

Comment:
I did not notice this before, but I guess this can use `access_multiuse_invite_by_id` function which will do the access checks except the creation permission check.